.github/workflows/main.yml

```
@@ -86,19 +88,24 @@ jobs:
           git clone --branch v1.5.0 https://github.com/JuulLabs-OSS/mcuboot
```

Upgrading the version avoid having to install cbor manually:
```suggestion
          git clone --branch v1.7.2 https://github.com/JuulLabs-OSS/mcuboot
```

(See https://github.com/JF002/InfiniTime/pull/534/files#r678790633)